### PR TITLE
Revert "Use MLIR op lowering of FusedBatchNormV3 and FusedBatchNormGradV3 for tf2xla"

### PR DESCRIPTION
**Pull request by trevor-m**
Reverts commit https://github.com/tensorflow/tensorflow/commit/676af55749f95c486c6fce5b8b550c5412e28a5b

We found via git bisect that this commit breaks convergence (loss goes to NaN) for a TF Keras Resnet50 model with XLA enabled.

### CODE REVIEW

**Comment by keerthanakadiri** [comment]
Hi @jpienaar , Can you please review this PR? Thank you .

**Comment by keerthanakadiri** [comment]
Hi @jpienaar , Can you please review this PR? Thank you .

**Comment by keerthanakadiri** [comment]
Hi @jpienaar , Can you please review this PR? Thank you .

**Comment by keerthanakadiri** [comment]
Hi @jpienaar , Can you please review this PR? Thank you !

**Comment by keerthanakadiri** [comment]
Hi @jpienaar , Can you please review this PR? Thank you !

**Comment by keerthanakadiri** [comment]
Hi @jpienaar , Can you please review this PR? Thank you !

**Comment by keerthanakadiri** [comment]
Hi @jpienaar , Can you please review this PR? Thank you !

**Comment by keerthanakadiri** [comment]
Hi @jpienaar , Can you please review this PR? Thank you !

**Comment by keerthanakadiri** [comment]
Hi @jpienaar , Can you please review this PR? Thank you !

**Comment by keerthanakadiri** [comment]
Hi @MichaelHudgins  , Can you please review this PR? Thank you !

**Comment by mihaimaruseac** [comment]
Closing PRs that have been stale for over a year. Please open again if they still relevant and we will attempt to find faster reviewers.
